docs/integrations/sources/zoom-lc-cdk.md

```diff
@@ -0,0 +1,61 @@
+# Zoom
+
+## Overview
+
+
+The following connector allows airbyte users to fetch various meetings & webinar data points from the [Zoom](https://zoom.us) source. This connector is built entirely using the [low-code CDK](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview/).
+
```

Comment:
Please remove the or replace the other Zoom documentation. I'll remove the singer code in the future.

Reply:
Done. Thank you.